helper.js

```diff
@@ -201,7 +201,7 @@ HELPER.compile = (() => {
 	const Autoprefix = ( file ) => {
 		const data = Fs.readFileSync( file, 'utf-8');
 
-		Postcss([ Autoprefixer ])
+		Postcss([ Autoprefixer({ browsers: ['last 2 versions', 'ie 8', 'ie 9', 'ie 10'] }) ])
```

Comment:
The user won't have control over this bit. You're confusing this with [pancake](https://github.com/govau/pancake). This helper only does some compiling for us now until pancake takes the cake... haha